collectors/ebpf.plugin/ebpf.c

```diff
@@ -712,10 +712,9 @@ static void parse_port_list(void **out, char *range)
     }
 
     if (first > last) {
-        info("The specified order [%d, %d] is inverted, Netdata is fixing it!", first, last);
-        int tmp = first;
-        first = last;
-        last = tmp;
+        info("The specified order %s is wrong, the smallest value is always the first!", copied);
```

Comment:
do we have `warning`?

Reply:
No, we have `error`, `fatal` and `info`.

Reply:
@mfundul i remember there was a talk about adding `warn`, i dont really understand why we dont still have it. Any ideas? What stops us? 😄